### ISSUE TITLE
Blender 4 compatibility

### ISSUE THREAD

**Issue by bartteunis** [blender-to-gmstudio/blender-gms-vtx-buffer#38]
Check if there have been any changes to Blender's `bpy` API and change what's needed/required.

**Comment by Gizmo199** [comment]
Not sure if its related, but I get an error similar to the one posted [in this reddit thread](https://www.reddit.com/r/blender/comments/1gunag7/keyshot_plugin_error_mesh_object_has_no_attribute/). When trying to export I get a the `object has no attribute 'calc_normals_split'` error and it will not export, even when only trying to export basic position and UVs

**Comment by bartteunis** [comment]
Yeah, it is the exact same issue. The way split normals are calculated changed a bit in Blender 4.1 and the exporter is trying to call a function that no longer exists in that Blender version (and subsequent versions). This was fixed already in the SMF exporter (https://github.com/blender-to-gmstudio/blender-to-smf/issues/52) and I changed it here as well now. This [new release](https://github.com/blender-to-gmstudio/blender-gms-vtx-buffer/releases/tag/v1.0.14) shouldn't have that issue anymore.
